lib/swaps/SwapClient.ts

```diff
@@ -369,6 +372,9 @@ abstract class SwapClient extends EventEmitter {
   public isOutOfSync(): boolean {
     return this.status === ClientStatus.OutOfSync;
   }
+  public isNoInvoiceSupport(): boolean {
```

Comment:
Let's call this method `hasNoInvoiceSupport`
```suggestion
  public hasNoInvoiceSupport(): boolean {
```